owasp-zap/authenticator/Gruntfile.js

```diff
@@ -0,0 +1,64 @@
+//
+//  Gruntfile.js
+//  CompactConnect
+//
+//  Created by InspiringApps on 7/22/2024.
```

Comment:
Do we want to have a reference here to InpsiringApps? My understanding was we usually don't want the company name anywhere in the code base.

Reply:
That's following the example set in `webroot`, but I agree - I'll remove it.